src/scripts/install.sh

```diff
@@ -72,7 +77,7 @@ if [[ $os = CYGWIN* || $os = MINGW* || $os = MSYS* ]]; then
     mpmdir=$(cygpath "$mpmdir")
     batchdir=$(cygpath "$batchdir")
 elif [[ $os = Darwin ]]; then
-    if [[ $arch == arm64 ]]; then
+    if [[ $arch = arm64 && ! $mpmrelease < "r2023b" ]]; then
```

Comment:
The fact that `=` is POSIX for checking for equality always throws me for a loop